tests/test_service_resources.py

```diff
@@ -40,7 +40,12 @@ def test_error_result_reports_error_on_success(self):
 
     def test_invalid_json_reports_error_on_success(self):
         self.mock_ok_service(data=None)
+        json_decoding_error_messages = {
+            'python2_error': 'No JSON object could be decoded',
+            'python3_error': 'Expecting value: line 1 column 1 (char 0)',
+        }
+
         result = self.service.check()
         self.assertEqual(result['status'], ERROR)
         self.assertEqual(result['result'], None)
-        self.assertEqual(result['error'], 'No JSON object could be decoded')
+        self.assertIn(result['error'], json_decoding_error_messages.values())
```

Comment:
> handle the decode error [..] as a special case, we can guarantee a consistent error message [..], right?
I'm glad you point that out!  I had considered doing that, but this was simpler and so I started here.  I think you are right; a consistent error message across Python versions would be preferable.

----

That was an interesting [article](http://blog.jayfields.com/2008/02/testing-expect-literals.html) you shared.

> To be clear, the expected value should be the literal itself, not a variable holding a literal that was created earlier in the test.
I have been of the opposite mind, but I think that he make some good points.  I'll give it a try for a while and see if I think it makes my life and my code better.